### ISSUE TITLE
Jest  Cannot find module 'react-use-clipboard'

### ISSUE THREAD

**Issue by wroughtec** [danoc/react-use-clipboard#3]
```
Test suite failed to run

    Cannot find module 'react-use-clipboard' from 'x.tsx'

    However, Jest was able to find:
        './x.tsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'jsx', 'ts', 'tsx', 'json'].
```

This issue arose in version 0.1.3 
0.1.2 had no issue

I assume its to do the the file extension change in 0.1.3 but you may know better.

Should point out to get around I have mocked it but raised this encase anyone else runs into it

**Comment by wroughtec** [comment]
hmm also seems eslint doesn't like it:
`error  Unable to resolve path to module 'react-use-clipboard'  import/no-unresolved`

**Comment by danoc** [comment]
Hey @wroughtec — Thanks for the issue. I'm not able to reproduce this one.

Can you see if the package exists in your `package.json`, `yarn.lock` or `package-lock.json`, and `node_modules` folder?

**Comment by wroughtec** [comment]
yes is in the lock file:
```
react-use-clipboard@0.1.3:
  version "0.1.3"
  resolved "https://registry.yarnpkg.com/react-use-clipboard/-/react-use-clipboard-0.1.3.tgz#46dfbdb08586d1335970a7fdd88383ec64b95dc4"
  integrity sha512-EDKKJpduJtqpEDLXPdqcoQszTGyctg8V7tO8DWPAUFDnZOvfVA/EfTCYzugrpRS3ikeFbBbR8MrBrHJ2DhhqGg==
  dependencies:
    copy-to-clipboard "^3.0.8"
```

Eslint config as below:

```
const path = require('path');

module.exports = {
  extends: [
    'plugin:cypress-dev/general',
    'plugin:cypress-dev/tests',
    'plugin:cypress-dev/react',
    'plugin:import/typescript',
    'plugin:@typescript-eslint/recommended',
    'airbnb',
    'prettier',
    'prettier/@typescript-eslint',
    'prettier/react'
  ],
  plugins: [
    'html',
    'import',
    'eslint-plugin-env',
    'react-hooks',
    'jasmine',
    'cypress',
    'chai-friendly',
    '@typescript-eslint',
    'prettier'
  ],
  globals: { google: true, analytics: true },
  parser: '@typescript-eslint/parser',
  parserOptions: {
    ecmaVersion: 2018,
    sourceType: 'module',
    ecmaFeatures: {
      jsx: true
    },
    tokens: true
  },
  env: {
    es6: true,
    browser: true,
    node: true,
    jest: true,
    jasmine: true,
    serviceworker: true,
    'cypress/globals': true
  },
  rules: {},
  settings: {
    'import/resolver': {
      'babel-module': {},
      typescript: {},
      node: true,
      'eslint-import-resolver-typescript': true
    }
  },
  overrides: [
    {
      files: ['**/__tests__/**'],
      settings: {
        'import/resolver': {
          jest: {
            jestConfigFile: path.join(__dirname, './jest.config.js')
          }
        }
      }
    },
    {
      files: ['**/__tests__/*.test.ts', '**/__tests__/*.test.tsx'],
      rules: {
        '@typescript-eslint/explicit-function-return-type': 0
      }
    }
  ]
};
```

jest config:

```
const path = require('path');

module.exports = {
  verbose: true,
  rootDir: './src',
  moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx', 'json'],
  transform: {
    '^.+\\.(js|jsx)$': 'babel-jest',
    '^.+\\.(ts|tsx)$': 'babel-jest'
  },
  transformIgnorePatterns: ['[/\\\\]node_modules/\\\\].+\\.(js|jsx|ts|tsx)$', '\\.(css|scss|sass|less)$'],
  coveragePathIgnorePatterns: ['./public/', './node_modules/'],
  setupFiles: ['./testsSetup/setupTests.js', 'jest-localstorage-mock'],
  globals: {
    TZ: 'UTC'
  },
  moduleDirectories: ['node_modules', 'src', 'testUtils', __dirname],
  collectCoverageFrom: [],
  reporters: ['default', 'jest-junit'],
  watchPlugins: ['jest-watch-typeahead/filename', 'jest-watch-typeahead/testname', 'jest-watch-select-projects'],
  setupFilesAfterEnv: ['./testsSetup/extendJest.js'],
  moduleNameMapper: {
    '\\.(css|scss|sass|less)$': 'identity-obj-proxy',
    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
      '<rootDir>/__mocks__/fileMock.js'
  }
};
```

Tried on both Mac and ubuntu same error rolling back to 0.1.2 resolves, its probably something in my config

**Comment by danoc** [comment]
@wroughtec — Not too sure what the issue is. Happy to help debug if there's a reproduction I can clone locally.

I just published version `0.1.4` which adopts TypeScript. It's possible that the new version accidentally also fixes the issue you're running into.

**Comment by wroughtec** [comment]
Yep fixed my issue 👍
